jekyll/_cci2/language-android.md

```diff
@@ -24,128 +24,50 @@ This guide provides an introduction to Android development on CircleCI.
 If you are looking for a `.circleci/config.yml` template for Android,
 see the [Sample Configuration](#sample-configuration) section of this document.
 
-**Note:**
-We now have an Android machine image available in preview on CircleCI Cloud that supports x86 Android emulators and nested virtualization. Documentation on how to access it is available [here](https://github.com/CircleCI-Public/android-image-preview-docs).
-Another way to run emulator tests from a job is to consider using an external service like [Firebase Test Lab](https://firebase.google.com/docs/test-lab).
-For more details,
-see the [Testing With Firebase Test Lab](#testing-with-firebase-test-lab) section below.
+**Note:** We now have an Android machine image available in preview on CircleCI
```

Comment:
Shall we remove the `in preview` part? (I'm assuming this PR will be merged after the image is in GA status)